velodrome/transform/influx.go

```diff
@@ -56,6 +65,11 @@ func (config *InfluxConfig) CreateDatabase(tags map[string]string) (*InfluxDB, e
 		return nil, err
 	}
 
+	err = dropSeries(client, measurement, config.DB, tags)
```

Comment:
Only when you want to update that specific metric, I'm tempted to say about once a month. Depends on the metric, between 10 seconds and 2 minutes.

It's honestly much better than having to connect to the node to manually remove the series ...